app/presenters/two_factor_authentication/sign_in_selection_presenter.rb

```diff
@@ -2,7 +2,7 @@ module TwoFactorAuthentication
   class SignInSelectionPresenter
     include ActionView::Helpers::TranslationHelper
 
-    attr_reader :configuration, :user
+    attr_reader :configuration, :user, :method
```

Comment:
i wouldn't set the method as attr_reader for sign in selection presenter. maybe we can set method for attr_reader for sign_in_phone_selection_presenter

Reply:
That works! Thanks.